server/sonar-server/src/main/resources/org/sonar/server/component/ws/search_projects-example.json

```diff
@@ -14,15 +14,17 @@
       "tags": [
         "finance",
         "java"
-      ]
+      ],
+      "visibility": "public"
     },
     {
       "organization": "my-org-key-1",
       "id": "AU-TpxcA-iU5OvuD2FLz",
       "key": "another_project",
       "name": "My Project 2",
       "isFavorite": false,
-      "tags": []
+      "tags": [],
+      "visibility": "public"
```

Comment:
could be interesting in the example to have at least once the other possible value : "private"